test/refactoring/ConvertAsync/labeled_closure_params.swift

```diff
@@ -0,0 +1,59 @@
+// RUN: %empty-directory(%t)
+
```

Comment:
Worth adding some convert call/function tests or do you think these cover it well enough?

Reply:
More test cases are always a good idea. I added one. The refactored code doesn’t change because we are just binding the tuple as `let (a, b) = callToAsyncFunc()` which also works if the returned tuple is labeled.